Core/Src/main.c

```diff
@@ -203,6 +203,11 @@ osMutexId_t m_state_parameter_mutexHandle;
 const osMutexAttr_t m_state_parameter_mutex_attributes = {
   .name = "m_state_parameter_mutex"
 };
+/* Definitions for inverter_can_timer */
+osTimerId_t inverter_can_timerHandle;
+const osTimerAttr_t inverter_can_timer_attributes = {
+  .name = "inverter_can_timer"
+};

```

Comment:
Definir isso pelo cube. Por organização, mas mais importante, esse código será apagado quando um novo código for gerado, então usar o cube pra isso

Reply:
Bom disso que vc vai poder criar o timer pelo cube e nem vai precisar apagar essas linhas rs